js/bitfinex.js

```diff
@@ -468,7 +468,7 @@ module.exports = class bitfinex extends Exchange {
         let orderType = order['type'];
         let exchange = orderType.indexOf ('exchange ') >= 0;
         if (exchange) {
-            [ , orderType ] = order['type'].split (' ');
+            [ prefix, orderType ] = order['type'].split (' ');
```

Comment:
still need the "let" in ES7 strict mode afaik )

Reply:
Oh, `let` there is dangerous:
1) `orderType` is defined two lines above
2) its value won't go outside the next line which is the end of the block

But I missed a fact that `prefix` was not defined before, `let` is required, obviously.

So destructuring can't be used here at all, that's why @Deathamns removed it in his fix.

Reply:
> Btw, the transpiler now supports `const` as well ;)

I love `consts`! I haven't seen you used them in other places so decided you probably dislike them like you semicolons do :D

Ok, I'll start using them and we'll be able to add some rules to the linter.